_unittest/test_26_emit.py

```diff
@@ -898,6 +898,143 @@ def test_availability_1_to_1(self):
         assert len(radiosTX) == 2
         assert len(radiosRX) == 2
 
+    @pytest.mark.skipif(
+        config["desktopVersion"] <= "2023.1" or is_ironpython,
+        reason="Skipped on versions earlier than 2023.2",
+    )
+    def test_interference_script(self):
+        sys.path.append("examples/07-EMIT")
+        from interference_classification import interference_type_classification
+        from interference_classification import protection_level_classification
+
+        self.aedtapp = BasisTest.add_app(self, application=Emit)
+        # place components
+        rad1 = self.aedtapp.modeler.components.create_component("Bluetooth Low Energy (LE)")
+        ant1 = self.aedtapp.modeler.components.create_component("Antenna")
+        if rad1 and ant1:
+            ant1.move_and_connect_to(rad1)
+        rad2 = self.aedtapp.modeler.components.create_component("GPS Receiver")
+        ant2 = self.aedtapp.modeler.components.create_component("Antenna")
+        if rad2 and ant2:
+            ant2.move_and_connect_to(rad2)
+        rad3 = self.aedtapp.modeler.components.create_component("WiFi - 802.11-2012")
+        ant3 = self.aedtapp.modeler.components.create_component("Antenna")
+        if rad3 and ant3:
+            ant3.move_and_connect_to(rad3)
+
+        # Reduce the bluetooth transmit power
+        bands = rad1.bands()
+        for band in bands:
+            band.set_band_power_level(-20)
+
+        # Disable L2 P(Y) GPS Band
+        bands = rad2.bands()
+        for band in bands:
+            for child in band.children:
+                if "L2 P(Y)" in band.node_name:
+                    band.enabled = True
+                else:
+                    band.enabled = False
```

Comment:
Do you want to disable or enable "L2 P(Y)"?

Reply:
It's supposed to be enabled, I'll adjust the comments above it